Pipelines/Templates/deploy-Solution.yml

```diff
@@ -285,3 +285,15 @@ steps:
     testAccountPassword: '$(TestAutomationPassword)'
     makerPortalUrl: '$(TestAutomationMakerPortalUrl)'
     loginMethod: '$(TestAutomationLoginMethod)'
+
+- template: Hooks\deploy-portal-pre-hook.yml
+
+- task: microsoft-IsvExpTools.PowerPlatform-BuildTools.upload-paportal.PowerPlatformUploadPaportal@0
+  displayName: 'Upload Portal ${{parameters.solutionName}}'
+  inputs:
+    authenticationType: PowerPlatformSPN
+    PowerPlatformSPN: '${{parameters.serviceConnectionName}}'
+    UploadPath: $(ArtifactDropPath)\Portals\${{parameters.solutionName}}
+    Environment: '${{parameters.serviceConnectionUrl}}'
```

Comment:
@mawasile Can we include Condition (i.e., WebsiteID exists) for this Task? FYI @mikefactorial